test/OpenTelemetry.Tests/Metrics/MetricOverflowAttributeTestsBase.cs

```diff
@@ -215,7 +215,7 @@ public void MetricOverflowAttributeIsRecordedCorrectlyForCounter(MetricReaderTem
         counter.Add(15); // Record another measurement for zero tags
 
         // Emit 2500 more newer MetricPoints with distinct dimension combinations
-        for (int i = 2000; i < 4500; i++)
+        for (int i = 2002; i < 4502; i++)
```

Comment:
nit: Why have the discontinuity? 😄

```suggestion
        for (int i = 2001; i < 4501; i++)
```